pallets/detach/src/lib.rs

```diff
@@ -13,8 +13,11 @@ mod weights;
 
 use sp_core::{crypto::KeyTypeId, ecdsa, ed25519, H160, U256};
 
+
 /// Defines application identifier for crypto keys of this module.
 ///
+/// ¿
```

Comment:
This is some internal thing of substrate, it's fine to not document it on our codebase actually